src/docsfy/api/projects.py

```
@@ -0,0 +1,1592 @@
+    """
+    from urllib.parse import urlparse
+
+    try:
+        parsed = urlparse(url)
+        hostname = parsed.hostname
+        # Handle SSH format: git@hostname:org/repo.git
+        if not hostname and url.startswith("git@"):
+            try:
+                hostname = url.split("@")[1].split(":")[0]
+            except (IndexError, ValueError):
+                pass
```

Ruff (0.15.6)

[warning] Use `contextlib.suppress(IndexError, ValueError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(IndexError, ValueError): ...`

(SIM105)